src/table/pagedTable/pagedTable.tsx

```diff
@@ -36,13 +36,18 @@ export interface IPagedTableHook<T> {
 // context for our data table
 // can be used by nested components to for a reload of the data, for instance when editing/deleting rows
 
-export const PagedDataContext = React.createContext<IPagedDataContext>({ reloadData: () => {} })
+export const PagedDataContext = React.createContext<IPagedDataContext>({
+  reloadData: () => {
+    return undefined
```

Comment:
Just curious, why can't it be a `void` function?

Reply:
eslint throws an error that it's an unexpected empty func

Reply:
I see